src/notation/qml/MuseScore/NotationScene/internal/TextStylePopup.qml

```diff
@@ -0,0 +1,298 @@
+                            title:qsTrc("inspector", "Align bottom"),
+                            description: qsTrc("inspector", "Align bottom edge of text to reference point")
+                        },
+                        {
+                            iconRole: IconCode.TEXT_ALIGN_BASELINE,
+                            typeRole: TextTypes.FONT_ALIGN_V_BASELINE,
+                            title: qsTrc("inspector", "Align baseline"),
+                            description: qsTrc("inspector", "Align baseline of text to reference point")
+                        }
+                    ]
+
+                    delegate: FlatRadioButton {
+                        toolTipTitle: modelData.title
+                        toolTipDescription: modelData.description
+
+                        width: 30
+                        transparent: true
+
+                        iconCode: modelData.iconRole
+                        checked: false //currentVerticalAlignment(/*TODO*/)
+                        onToggled: {
+                            //TODO
+                        }
+                    }
+                }
+            }
+            RowLayout { //Column 2, Row 2
+                RadioButtonGroup {
+                    id: subscriptOptionsButtonList
+                    height: 30
+
+                    model: [
+                        { iconRole: IconCode.TEXT_SUBSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_BOTTOM, titleRole: qsTrc("inspector", "Subscript") },
+                        { iconRole: IconCode.TEXT_SUPERSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_TOP, titleRole: qsTrc("inspector", "Superscript") }
+                    ]
+
+                    delegate: FlatRadioButton {
+
+                        width: 30
+                        transparent: true
+
+                        iconCode: modelData["iconRole"]
+                        checked: false //currentSubscriptOption(/*TODO*)
+                        onClicked: {
+                            //TODO
+                        }
+                    }
+                }
+                StyledTextLabel {
+                    text: qsTrc("inspector", "Line spacing:")
+                }
+                IncrementalPropertyControl {
+                    id: lineSpacingSpinBox
+
+                    Layout.preferredWidth: 60
+
+                    isIndeterminate: false //TODO
+                    currentValue: 0 //TODO
+
+                    onValueEditingFinished: function(newValue) {
+                        //TODO
+                    }
+                }
+            }
+        }
+        RowLayout {
+            FlatButton {
+                Layout.fillHeight: true
+                icon: IconCode.FLAT
+                text: qsTrc("inspector", "Add symbols")
+            }
+
+            ColumnLayout {
+                FlatButton {
+                    width: 120
+                    text: qsTrc("inspector", "Text Style")
```

Comment:
Our style is to use `Text style` with lowercase `s`. This style is referred to as "Sentence case". 
The style of capitalising every word (which we _don't_ do) is called "Title Case".
```suggestion
                    text: qsTrc("inspector", "Text style")
```